docs/standard-input-output.adoc

```diff
@@ -0,0 +1,64 @@
+:page-title: Standard input/output
+:page-description: JUnit Jupiter extension for simulating standard input or catching standard output.
+
+The `StdIOExtension` adds a simple way to test your classes that read from the standard
+input (`System.in`) or write to the standard output (`System.out`).
+
+== How to use the extension
+
+If you would like to use the special parameters provided by the extension
```

Comment:
i just want to raise a discussion here, about our documentation style, or more about unifying it - and maybe find a guideline on how to write docs. Currently it seems like we have two different approaches:

1. the personal/informal one, in which we address the developer and his work directly with you and your tests (like here)
2. the formal one, in which it seems like we try to avoid such constructs (eg. RangeSource)

For a good comprehensive documentation i think we should agree on one style to use. As it makes reading the docs easier to grasp and shows some kind of consistency. The questions are

- is this something which only bothers me? than lets continue, as i did not say anything :)
- what would be the general preference. I tend to use the formal one, simply because it feels more 'technical' to me.

What is the general take on this?

Reply:
Good spot, but I won't discuss that here, as we most probably have to align multiple places. So I'll create an bishue for this.